pages/common/firebase.md

```diff
@@ -0,0 +1,24 @@
+# firebase
+
+> Firebase Command Line Interface (CLI) Tools.
+> Used to test, manage, and deploy your Firebase project from the command line. More information: <https://www.npmjs.com/package/firebase-tools>.
```

Comment:
I think the offical repository is better because you can install this tool without npm.
```suggestion
> Used to test, manage, and deploy your Firebase project from the command line.
> More information: <https://github.com/firebase/firebase-tools>.
```